### PR TITLE
Remove potentially offensive words, capitalization

### PR DESCRIPTION
**Pull request by gimmerain4days**
Commented out some words for easier review and reinstatement. 
Capitalized some words that should be capitalized. 
Removed or commented on some abbreviations.
Added comments to words removed and some not removed. If there is a criteria for inclusion/exclusion, words might be found more quickly. (i.e. remove all demonyms)

### CODE REVIEW

**Comment by gimmerain4days** [comment]
looks like .json doesn't like comments. I think they are useful, though. I am open to any suggestions.

**Comment by Bruception** [comment]
> looks like .json doesn't like comments. I think they are useful, though. I am open to any suggestions.

No comments please

File: frontend/static/languages/english_25k.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "learner",
     "timekeeper",
     "muddle",
-    "scrotum",
+    //"scrotum",//Potentially Offensive
```
Comment (Bruception) [inline frontend/static/languages/english_25k.json:13]: Looks like you are still trying to include comments.

Comment (gimmerain4days) [reply frontend/static/languages/english_25k.json]: Just updated to remove the comments.

**Review by Miodec — APPROVED** [review]
Thanks

**Pull request merged**